src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	public final String value;
```

Comment:
Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.